src/Elemental/View/Form/Field/Switch.elm

```diff
@@ -21,7 +21,9 @@ type alias Options msg =
     , text : String
     , disabled : Bool
     , size : Size
-    , verticalPadding : Size -> Float
+    , spacerMultiples :
+        { y : Size -> Float
+        }
```

Comment:
This should be a part of the `Theme`

Reply:
Any preference on the name of that spacer?

Reply:
Do you mean the name of the spacer in between the handle and text? I don't have a preference for that :)

`spacerMultiples` however, can be included as a property inside the `Theme` record directly (see https://github.com/realfolk/elm-elemental/blob/master/src/Elemental/View/Form/Field/Select.elm#L62)